vulcan/lib/server/workflows/bar-plot.metadata.json

```diff
@@ -1,8 +1,8 @@
+  "lastModified": "Jan 7, 2022",
+  "vignette": "# Bar Plot Workflow\n\nThis workflow allows users to create a composition-focused bar plot from data of their choice. At present, that data must come from a 'query' of magma data, but we hope to provide other options in the future.\n\n## Selecting Data via Timur-Query:\n\nThis Data Loading method uses the 'Data Source magma query' Primary Inputs group. The easiest way to establish the proper inputs for querying from magma is to utilize the Timur-Query page. You can use Timur-Query's own documentation to learn how to set up that page to query the data you want. After setting up your query targets there: \n\n- 'query input': Copy here the 'query:' portion of the Query page's yellow box.\n- 'user_columns input': Copy here the 'user_columns:' portion of Query page's yellow box.\n- 'expand_matrices input': [Recommendation = leave as is.] This input defaults to True both here and in the Query page (where it exists as a slider button below the yellow box). Normally, you will want to leave it True as the non-expanded format is less accessible to the visualization machinery here.\n\n## Generating the plot\n\nAfter data is obtained, a new user input widget will show up in which you can select which columns to use for the x axis groupings, the data whose composition within each group should summarized in the y axis, and whether to scale the resulting bars by percentage among each group ('fraction') or absolute counts ('counts'), as well as other advanced options. NOTE: The x and y axis data for a bar plot must be a discrete (not continuous) data type.\n\n#### We plan to make the Timur-Query -> Vulcan workflow transfer process simpler in the future!",
```

Comment:
I would probably call the query and user_columns box "green" rather than yellow...but understandably that may differ by viewer...we should probably label that section in Query, to make it more accessible.

Reply:
`each group should summarized` -> `each group should be summarized`?

Reply:
lol I should have checked the color with someone "color-typical"!

Made these fixes.